<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>477</width>
    <height>572</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="4" column="1" colspan="3">
     <widget class="QLabel" name="resultValueLabel">
      <property name="text">
       <string>0.0</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QLabel" name="upperBoundMaxVal">
      <property name="text">
       <string>100</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QSlider" name="upperBoundSlider">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="upperBoundMinVal">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="lowerBoundMinVal">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="lowerBoundLabel">
      <property name="text">
       <string>Lower bound</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2" alignment="Qt::AlignHCenter|Qt::AlignTop">
     <widget class="QLabel" name="lowerBoundCurVal">
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLabel" name="lowerBoundMaxVal">
      <property name="text">
       <string>100</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="upperBoundLabel">
      <property name="text">
       <string>Upper bound</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QSlider" name="lowerBoundSlider">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="resultLabel">
      <property name="text">
       <string>Result</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2" alignment="Qt::AlignHCenter|Qt::AlignTop">
     <widget class="QLabel" name="upperBoundCurVal">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>477</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
